### DISCUSSION TITLE
2023/08/lan-ke-shan/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#90] [General]
# 2023/08/lan-ke-shan/

https://xiangyun.rbind.io/2023/08/lan-ke-shan/

**Comment by XiangyunHuang** [comment]
这周末预定去长安大剧院看昆曲《牡丹亭》和《李清照》，前者心心念念很久了，后者还蛮好奇的，为啥李清照出现会在昆曲里了呢？

**Reply by XiangyunHuang** [reply]
竟不曾想《烂柯山》是最近听过的感染力最强的一曲。《西厢记》、《牡丹亭》和《李清照》也是悲剧，但处理得比较热闹，舞台也非常漂亮。《西厢记》和《牡丹亭》现场观众比较多。几场下来，我以为首先看名角，其次是曲目。若名角和名曲凑一块了，那一票难求。若有名角在，任何曲目都值得一听。

**Reply by XiangyunHuang** [reply]
还有一个感受，昆曲好像不能连续地去听，因为自己了解的那点东西太浅了，再听听不出更多的韵味了，所以，如果想获得更多的感受，应该继续加深对昆曲的了解，这可太难了。
